src/components/editor/editor.tsx

```diff
@@ -15,12 +15,14 @@ interface EditorProps {
   updateChange: (note) => void;
   auth: any;
   projectID: number | string;
+  userId: number | string;
```

Comment:
try to be more consistent with the variable names
```suggestion
  userID: number | string;
```